modules/core/src/main/java/io/fluo/webindex/core/DataUtil.java

```diff
@@ -22,6 +22,9 @@
 
 public class DataUtil {
 
+  public static String URL_SEP_REGEX = "[\\/\\?\\#]";
```

Comment:
Was not sure escaping was needed in regex char set.  Also I was not sure if would honor escaping or treat `\` as a character in the set.  I wrote the following program to see.

```
    String re1 = "[\\/\\?\\#]";
    String re2 = "[/?#]";

    String data = "one/two?three#four\\five";

    System.out.println(Arrays.asList(data.split(re1)));
    System.out.println(Arrays.asList(data.split(re2)));
```

It printed 

```
[one, two, three, four\five]
[one, two, three, four\five]
```